### PR TITLE
[TW-867] Collection link and guest role

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Explained how to allow Guests to view collections using generated link
* Explained the team-level Guest role
* Explained how users with the Guest role can request elevated permissions
* Explained how Team Admins can elevate Guest role permissions
* Updated screenshots

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
Changes look great! 🌟

**Review by ArjunSingh-PM — COMMENTED** [review]
The changes look good. I have added a few comments that need to be addressed.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -140,6 +141,16 @@ For full details about the Postman team roles that you can assign as a Team Admi
 
 You can manage external partners similarly to internal team members. To learn more, see [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/).
 
+### Managing guests
+
+You can manage external users who have permission to view specific collections and send requests in the collections. External users with these permissions are assigned the [Guest role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles). A Team Admin can change an external user's role to the Developer role by selecting **Convert**. When you change an external user's role from the Guest role to the Developer role, they are added to the team and get access to all team resources and workspaces.
+
+![Convert Guest role](https://assets.postman.com/postman-docs/v10/convert-guest-role-v10.jpg)
+
+> Your team must have [available seats](/docs/administration/billing/#changing-your-plan) or [Auto-Flex enabled](/docs/administration/billing/#using-auto-flex) to change an external user's role from the Guest role to the Developer role.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md:150]: It's my understanding that your team must have available seats if you want to upgrade/elevate the user from Guest role --> Developer role. This is true, right?

I can reword the sentence if this isn't clear. Thanks!

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ You can [assign](/docs/administration/managing-your-team/managing-your-team/) on
...
 * **Partner Manager** (Internal, [Enterprise plans only](https://www.postman.com/pricing)) - Manages all [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/) within an organization. Controls Partner Workspace settings and visibility, and can send invites to partners. To learn more, see [Partner roles](#partner-roles).
 * **Partner** (External, [Enterprise plans only](https://www.postman.com/pricing)) - All partners are automatically granted the Partner role at the team level. Partners can only access the [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/) they've been invited to. To learn more, see [Partner roles](#partner-roles).
+* **Guest** (External) - Guests can view collections and send requests in the collections that have been shared with them. This role can't be directly assigned to a user. To learn more, see [Allowing external users to view collections](/docs/collaborating-in-postman/sharing/#allowing-external-users-to-view-collections).
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md]: This is now updated.

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 3]
```diff
@@ -109,20 +111,57 @@ Anyone with this link has read-only access to the collection's JSON. The data th
...
+
+### Changing external user access to collections
+
+You can change an external user's role at the team and collection levels. You can also revoke access to viewing collections for new and existing external users.
+
+To learn how to change an external user's team role, see [Managing guests](/docs/administration/managing-your-team/managing-your-team/#managing-guests). To learn how an external user can request a role change, see [Requesting Editor role access for a collection as an external user](/docs/collaborating-in-postman/requesting-access-to-collections/#requesting-editor-role-access-for-a-collection-as-an-external-user).
+
+You can turn off the toggle next to **Allow Guests with link to view collection**. When you turn this off, external users can no longer be assigned the Guest role at the team level and the limited Viewer role at the collection level. External users already assigned the Guest role at the team level and the limited Viewer role at the collection level will continue to have access to the collection.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md]: This is now updated.

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 4]
```diff
@@ -109,20 +110,41 @@ Anyone with this link has read-only access to the collection's JSON. The data th
...
 
 > If your team has [SSO enabled](/docs/administration/sso/intro-sso/), external users will be required to sign in using your team's SSO.
+
+## Allowing external users to view collections
+
+You can allow external users who aren't in your Postman team to view a specific collection and send requests in that collection. Once you allow external users to view a specific collection, send them the collection URL to view the collection.
+
+1. From the collection you want to share, select <img alt="Share icon" src="https://assets.postman.com/postman-docs/icon-share.jpg#icon" width="16px"> **Share**.
+1. Select **With people**.
+1. Turn on the toggle next to **Allow Guests with link to view collection**.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md]: It's my understanding that we changed the toggle switch text to "Allow Guests with the link to join your team and view collection". I updated occurrences of this text and the screenshot that includes it.

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 5]
```diff
@@ -140,6 +141,18 @@ For full details about the Postman team roles that you can assign as a Team Admi
...
+
+A Partner Manager can also view the workspaces a user assigned the Guest role can access. Go to your [team dashboard](https://go.postman.co/settings/team/members), and select **View resources**.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:152]: An Admin can view this, not a Partner Manager.

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 6]
```diff
@@ -109,20 +111,59 @@ Anyone with this link has read-only access to the collection's JSON. The data th
...
+
+### Changing external user access to collections
+
+You can change an external user's role at the team and collection levels. You can also revoke access to viewing collections for new and existing external users.
+
+To learn how to change an external user's team role, see [Managing guests](/docs/administration/managing-your-team/managing-your-team/#managing-guests). To learn how an external user can request a role change, see [Requesting Editor role access for a collection as an external user](/docs/collaborating-in-postman/requesting-access-to-collections/#requesting-editor-role-access-for-a-collection-as-an-external-user).
+
+You can turn off the toggle next to **Allow Guests with the link to join your team and view collection**. When you turn this off, new and existing external users can no longer view the collection. If this is turned off for all collections an external user can view, they will be removed from the team. To add them back to the team, see [Allowing external users to view collections](#allowing-external-users-to-view-collections).
```
Comment (ArjunSingh-PM) [inline src/pages/docs/collaborating-in-postman/sharing.md:157]: Existing guests in the team will continue to be able to access the collections that they have been assigned (i.e. the collections that were shared with them) even if the toggle is turned off.

No new guests will be added to the team using a collection link that does not have the toggle turned on.

**Review by ArjunSingh-PM — COMMENTED** [review]
Added comments for changes to be made

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 7]
```diff
@@ -140,6 +141,18 @@ For full details about the Postman team roles that you can assign as a Team Admi
 
 You can manage external partners similarly to internal team members. To learn more, see [Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/).
 
+### Managing guests
+
+You can manage external users who have permission to view specific collections and send requests in the collections. External users with these permissions are assigned the [Guest role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles). A [Partner Manager](/docs/collaborating-in-postman/roles-and-permissions/#partner-roles) can change an external user's role to the Developer role by going to your [team dashboard](https://go.postman.co/settings/team/members) and selecting **Convert**. When you change an external user's role from the Guest role to the Developer role, they are added to the team and get access to all team resources and workspaces.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md]: This is now updated.

File: src/pages/docs/collaborating-in-postman/sharing.md (hunk 6 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md]: This is now updated.

**Review by ArjunSingh-PM — APPROVED** [review]
Changes look good.

**Pull request merged**